### ISSUE TITLE
Steering committee composition

### ISSUE THREAD

**Issue by effigies** [nipreps/GOVERNANCE#6]
Should there be some rule that each actively maintained prep should have some representative on the steering committee? I'd suggest each project nominate a rep.

**Comment by oesteban** [comment]
I would generally recommend that the steering committee draws the candidates from the maintainers lists of projects, but not as a requirement.

However, I can see where this is coming from - you want the steering committee to be formed by people who keep tabs on relevant projects.

Instead, this could be achieved with the new technical monitoring meetings. Since this issue was open way way before we started these "IT" meetings, I guess the need for communication organically found its way.

Would you agree with these two items?

**Comment by oesteban** [comment]
What about this?

https://github.com/nipreps/GOVERNANCE/pull/20/files#diff-1e24a937d37436673f438d953367fbe0e175cde5bab7bca1df250495c7ad7d57R30

**Comment by effigies** [comment]
The concern was more that without representation from all of the active projects, decisions could be made to the benefit of represented projects without fully grasping the consequences for less well-represented projects.

That said, a structural safeguard like that may be more than we can currently supply the effort for.

**Comment by oesteban** [comment]
> That said, a structural safeguard like that may be more than we can currently supply the effort for.

Yes, this is my background thinking here. However, I believe the spirit of the original MVG project was to afford projects with independence so that the steering committee only intervenes when deciding whether a new "prep" can be accepted and in case of conflicts. I don't dislike the model (this is just a personal preference, we should find a consensus solution), so the current proposal tries to ensure the representation of projects through alternative means, such as the technical monitoring meetings and the "outbound coordinator" maintainer-role:

https://github.com/nipreps/GOVERNANCE/blob/fcb61078f80de6e693cad28622aaecae35fb116b/project-docs/.maint/HOW_WE_WORK.md?plain=1#L24-L25

In other words, I'm not opposed to ensuring a fair representation of projects at all. To do so, I think we need to find a less hierarchical way (than the steering committee) of implementing it.
